packages/flutter/test/widgets/banner_test.dart

```diff
@@ -295,4 +295,32 @@ void main() {
       areCreateAndDispose,
     );
   });
+
+  testWidgets('Banner widget respects shadow', (WidgetTester tester) async {
+    debugDisableShadows = false;
+    await tester.pumpWidget(
+      const Directionality(
+           textDirection: TextDirection.ltr,
+        child: Banner(
+            message: 'Shadow banner',
+            location: BannerLocation.topEnd,
+            shadow: BoxShadow(
+              color: Color(0xFF008000),
+              blurRadius: 8.0,
+            ),
+          )
+      ),
+    );
+    final Finder customPaint = find.byType(CustomPaint);
+
+    expect(customPaint, findsOneWidget);
+
+    final CustomPaint paintWidget = tester.widget(customPaint);
+    final BannerPainter painter = paintWidget.foregroundPainter! as BannerPainter;
+
+    expect(painter.shadow.color, const Color(0xFF008000));
+    expect(painter.shadow.blurRadius, 8.0);
+    
```

Comment:
(btw, it looks like [Linux Analyze](https://github.com/flutter/flutter/pull/155296/checks?check_run_id=30267715030) got mad about the whitespace here)

```suggestion

```

Reply:
Thanks for the help @nate-thegrate. Made the requested changes.